spec.html

```diff
@@ -27868,7 +27867,7 @@ <h1>The String Constructor</h1>
         <h1>String ( _value_ )</h1>
         <p>When `String` is called with argument _value_, the following steps are taken:</p>
         <emu-alg>
-          1. If no arguments were passed to this function invocation, let _s_ be `""`.
+          1. If _value_ is not present, set _value_ to `""`.
```

Comment:
Presumably that should be `... set _s_ ...`

Reply:
```suggestion
          1. If _value_ is not present, set _s_ to the empty String.
```